packages/cel/src/type.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright 2024-2025 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { DescMessage, MessageShape } from "@bufbuild/protobuf";
+import type { CelList, CelMap, CelUint } from "./value/value.js";
+
+/**
+ * Types of CEL values.
```

Comment:
Currently, the linter doesn't run on this package, which isn't great. It'll be difficult to enable it with a single change. What if we enable the linter (add a "lint" script to package.json), but ignore everything but new files for the transition period (exclude all, include type.ts, type.test.ts in [biome.json](https://biomejs.dev/guides/configure-biome/#control-files-via-configuration))?